patches/@polkadot-cloud+react+0.1.39.patch

```diff
@@ -0,0 +1,15 @@
+diff --git a/node_modules/@polkadot-cloud/react/overlay/Overlay/Modal.js b/node_modules/@polkadot-cloud/react/overlay/Overlay/Modal.js
+index 675baaa..40df904 100644
+--- a/node_modules/@polkadot-cloud/react/overlay/Overlay/Modal.js
++++ b/node_modules/@polkadot-cloud/react/overlay/Overlay/Modal.js
+@@ -134,9 +134,7 @@ export var Modal = function (_a) {
+                 },
+             }, style: { opacity: status === "opening" ? 0 : 1 }, children: _jsxs("div", { children: [_jsx(ModalHeight, { ref: heightRef, size: size, style: {
+                             height: modalHeight,
+-                            overflow: modalHeight >= modalMaxHeight && !(options === null || options === void 0 ? void 0 : options.disableScroll)
```

Comment:
`void 0`? 🤔 Is this automatically generated code?

Reply:
yes, I'm modifying the @polkadot-cloud/react library which seems to be already transpiled on npm :cry: